applications/GeoMechanicsApplication/python_scripts/geomechanics_analysis.py

```diff
@@ -214,6 +225,9 @@ def RunSolutionLoop(self):
                 for idx, node in enumerate(self._GetSolver().GetComputingModelPart().Nodes):
                     self._CalculateTotalDisplacement(node, old_total_displacements[idx])
 
+            for node in self._GetSolver().GetComputingModelPart().Nodes:
```

Comment:
Here, the calculation is done before `FinalizeSolutionStep`, while in the C++ route it's the other way around. Is there a specific reason? If not, I'd propose to make it consistent (and I think the python order is better, but I leave that up to you)

Reply:
Changed the C++ route, should now be consistent.